
{'exp_name': 'pedsim', 'exp_epi_len': 1000, 'pi_hidden_layers': [128, 128], 'pi_options': ['default', 'leaky-relu', 'default'], 'replay_size': 10000, 'gamma': 0.99, 'polyak': 0.995, 'alpha': 0.002, 'learning_rate': 0.0001, 'epochs': 1000, 'steps_per_epoch': 4000, 'batch_size': 512, 'n_log_epi': 10, 'start_steps': 1000, 'update_after': 4000, 'update_every': 1000, 'save_interval': 10, 'plot_rendering': True, 'seed': 1}
Traceback (most recent call last):
  File "sac_discrete.py", line 49, in <module>
    exp_obs_dim_, exp_act_dim_ = env_.get_dim()
  File "/home/jay/catkin_ws/src/social_navigation/scripts/gazebo_master.py", line 496, in get_dim
    o = self.reset()
  File "/home/jay/catkin_ws/src/social_navigation/scripts/gazebo_master.py", line 380, in reset
    self.prev_goal_distance = self.goal_distance
AttributeError: 'PedSim' object has no attribute 'goal_distance'
Traceback (most recent call last):
  File "sac_discrete.py", line 49, in <module>
    exp_obs_dim_, exp_act_dim_ = env_.get_dim()
  File "/home/jay/catkin_ws/src/social_navigation/scripts/gazebo_master.py", line 496, in get_dim
    o = self.reset()
  File "/home/jay/catkin_ws/src/social_navigation/scripts/gazebo_master.py", line 380, in reset
    self.prev_goal_distance = self.goal_distance
AttributeError: 'PedSim' object has no attribute 'goal_distance'
0.03188371658325195